README.md

```diff
@@ -15,6 +15,27 @@ with any kind of software without having to release its source code.
 Additionally, the example code has been released to the public domain (see the
 [separate license](examples/COPYING) for more information).
 
+Quick Started
+===============
+
+**Prerequisites** for building libgit2:
+
+1. [CMake](https://cmake.org/)
+2. [Python](https://www.python.org) is used by our test framework.
+3. C compiler
+   * Windows: Visual Studio is recommended
+   * Mac: Xcode is recommended
```

Comment:
Shouldn't we also declare that on Linux, it is fine if you use either GCC or Clang? Otherwise they might feel like third-class citizen here ;)